<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>433</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>424</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cloze Generator Options</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>411</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cb_ncf">
        <property name="toolTip">
         <string>Don't provide any context cues for first cloze item</string>
        </property>
        <property name="text">
         <string>No Context for First Item</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>&lt;b&gt;Handling of First and Last Items&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="cb_ncl">
        <property name="toolTip">
         <string>Don't provide any context cues for last cloze item</string>
        </property>
        <property name="text">
         <string>No Context for Last Item</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="cb_incr">
        <property name="toolTip">
         <string>For notes that have multiple clozes revealed per card,&lt;br&gt;gradually build up to full reveal count at the start,&lt;br&gt;and vice-versa in the end</string>
        </property>
        <property name="text">
         <string>Incremental Ends</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>411</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item row="2" column="0">
       <widget class="QSpinBox" name="sb_before">
        <property name="toolTip">
         <string>Number of context cues before the prompt</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>&lt;b&gt;Default Cloze Settings&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="sp_after">
        <property name="toolTip">
         <string>Number of context cues after the prompt</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="sp_cloze">
        <property name="toolTip">
         <string>Number of items to prompt for per card</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Context Before</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Cloze Length</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Context After</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Advanced</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>421</width>
       <height>306</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Remarks</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="le_og">
        <property name="toolTip">
         <string>Define a custom name for the field in question</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Sources</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLineEdit" name="le_fl">
        <property name="toolTip">
         <string>Define a custom name for the field in question</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;b&gt;Field Names&lt;/b&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="le_sc">
        <property name="toolTip">
         <string>Define a custom name for the field in question</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Original</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="le_tx">
        <property name="toolTip">
         <string>Define a custom name for the field in question</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="le_tt">
        <property name="toolTip">
         <string>Define a custom name for the field in question</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="le_st">
        <property name="toolTip">
         <string>Define a custom name for the field in question</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Text</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Full</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>&lt;b&gt;Custom Note Type&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="le_rk">
        <property name="toolTip">
         <string>Define a custom name for the field in question</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QLineEdit" name="le_model">
        <property name="toolTip">
         <string>Name of custom note type to use instead of the default one</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
